### ISSUE TITLE
Counter is going down instead of up

### ISSUE THREAD

**Issue by cesarferreira** [federicodesia/custom_timer#7]
on version `0.0.3` this worked fine but since I upgraded to `0.0.5` the counter goes down to negative values

```dart
CustomTimer(
      controller: _controller,
      from: Duration(seconds: 0),
      to: Duration(hours: 10),
      interval: Duration(milliseconds: 500),
      onBuildAction: CustomTimerAction.auto_start,
      builder: (CustomTimerRemainingTime remaining) {
        return Text(
          '${remaining.minutes}:${remaining.seconds}',
          style: _tema.title2().copyWith(color: _tema.backgroundTertiary()),
        );
      },
    )
```

**Comment by federicodesia** [comment]
The code automatically generated by flutter to migrate to null security in version 0.0.4 seems to modify this.
I will try to add unit tests to the widget so it doesn't happen again in the future.

You are a good tester and bug finder! 😆 I fixed it in version 0.0.6.
Thanks for helping!

**Comment by cesarferreira** [comment]
I can confirm it works, thanks!
